ansible/tasks/setup-supabase-internal.yml

```
@@ -63,3 +70,48 @@
     enabled: yes
     name: postgres_exporter
     daemon_reload: yes
+
+# supautils
+- name: supautils - download latest release
+  get_url:
+    url: "https://github.com/supabase/supautils/archive/refs/tags/v{{ supautils_release }}.tar.gz"
+    dest: /tmp/supautils-{{ supautils_release }}.tar.gz
+    checksum: "{{ supautils_release_checksum }}"
+
+- name: supautils - unpack archive
+  unarchive:
+    remote_src: yes
+    src: /tmp/supautils-{{ supautils_release }}.tar.gz
+    dest: /tmp
+  become: yes
+
+- name: supautils - build
+  make:
+    chdir: /tmp/supautils-{{ supautils_release }}
+  become: yes
+
+- name: supautils - install
+  make:
+    chdir: /tmp/supautils-{{ supautils_release }}
+    target: install
+  become: yes
+
+- name: supautils - set supautils.reserved_roles
+  become: yes
+  shell:
+    cmd: printf "supautils.reserved_roles = 'supabase_admin, supabase_auth_admin, supabase_storage_admin, dashboard_user, pgbouncer, service_role, authenticator, authenticated, anon'" >> /etc/postgresql/postgresql.conf
+
+- name: supautils - set supautils.reserved_memberships
+  become: yes
+  shell:
+    cmd: printf "supautils.reserved_memberships = 'pg_read_server_files, pg_write_server_files, pg_execute_server_program'" >> /etc/postgresql/postgresql.conf
+
+- name: supautils - set supautils.reserved_memberships
+  become: yes
+  shell:
+    cmd: printf "supautils.reserved_memberships = 'pg_read_server_files, pg_write_server_files, pg_execute_server_program'" >> /etc/postgresql/postgresql.conf
```

Seems there's a dup in these lines.

```suggestion
- name: supautils - set supautils.reserved_memberships
  become: yes
  shell:
    cmd: printf "supautils.reserved_memberships = 'pg_read_server_files, pg_write_server_files, pg_execute_server_program'" >> /etc/postgresql/postgresql.conf
```

---

Removed the duplicated line. Thanks!